R/utilities.R

```diff
@@ -73,8 +73,9 @@ check_names_extractable <- function(df_, col_name_df){
     }
 }
 
-error_on_date <- function(date, expected_date, days = 7, warning = FALSE){
-    fail <- abs(as.numeric(date - expected_date)) > days
+error_on_date <- function(date, expected_date, days = 30, warning = FALSE){
```

Comment:
does `date` need a default argument here? or is this inherited from the individual scrapers?

Reply:
the individual scrapers pass it in